live-examples/js-examples/array-some.html

```diff
@@ -0,0 +1,12 @@
+<pre>
+<code id='static-js'>var array = [1, 2, 3, 4, 5];
+
+var even = function(element, index, array) {
```

Comment:
You are not using `index` nor `array` inside the function so, no need to declare them as parameters.